### DISCUSSION TITLE
Image Sequence caching order

### DISCUSSION THREAD

**Discussion by AltaArts** [ggarra13/mrv2#257] [General]
We have just started using MRV2 in anger, and I could definitely be missing something.  But it seems that MRV2 caches the sequence starting from the last frame to the first?  Meaning when I load a sequence the blue bar starts at the right and loads to the left.

Is that something can be made to happen the other way?  With large files, it would be great to be able to start playing a sequence even though it may not be fully cached.

Thanks so much,

JRB.

**Comment by ggarra13** [comment]
I can verify the problem.  I put a fix for it for v1.2.0.  The problem appears seems to happen when:

- You have Preferences->Playback->Auto Playback Off
- Loop on

If you currently change any of these settings, you should be able to play right away.

If you can make it happen with some other settings, please open a bug report in the Issues section with the settings you have that make it cache incorrectly.

**Reply by AltaArts** [reply]
Nice find!  Yes, those are the settings I use, and turning on auto-playback does indeed make it cache forward.

Thanks for making the change.

JRB.
